pkg/operators/ebpf/params.go

```diff
@@ -25,7 +25,16 @@ import (
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/params"
 )
 
+type param struct {
+	*api.Param
+	fromEbpf bool
+	// Only valid for string parameters
+	strLen int
```

Comment:
It's the `strLen` field who is only valid for strings. The general `param` type is valid for all params. All those lines are added on this commit because it was moved from a different file.